contrib/idleconn-manpage/genallman.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Hi,

please rewrite so it's done in bourne shell. People run this on platforms that aren't linux/macosx. :)